packages/nimble-components/src/dialog/tests/dialog.spec.ts

```diff
@@ -254,7 +255,8 @@ describe('Dialog', () => {
         await disconnect();
     });
 
-    it('focuses the button with autofocus when the dialog opens', async () => {
+    // see https://github.com/ni/nimble/issues/1075
+    it('#failsFirefox focuses the button with autofocus when the dialog opens', async () => {
```

Comment:
Can you move the tag to the end of the test name instead of the start?

Reply:
Done